packages/database-mongo/src/mongo.ts

```diff
@@ -368,16 +368,34 @@ export class Mongo implements DatabaseInterface {
     );
   }
 
-  public async invalidateAllSessions(userId: string): Promise<void> {
-    await this.sessionCollection.updateMany(
-      { userId },
-      {
-        $set: {
-          valid: false,
-          [this.options.timestamps.updatedAt]: this.options.dateProvider(),
-        },
+  public async invalidateAllSessions(userId: string, excludeList?: string[]): Promise<void> {
+    let selector;
+    selector = { userId };
+
+    if (excludeList && excludeList.length > 0) {
+      let excludedObjectIds;
+      excludedObjectIds = excludeList;
+
+      if (this.options.convertSessionIdToMongoObjectId) {
+        excludedObjectIds = excludeList.map((sessionId) => {
+          return toMongoID(sessionId);
+        });
       }
-    );
+
+      selector = {
```

Comment:
to avoid conflict with future stuff we might add later the best would be to do `selector._id = ...`. That way we do not overwrite the previous object and can keep the old conditions